docs/getting_started/basics/exposing_components.md

```diff
@@ -49,6 +49,13 @@ For instance, the `"lsd:module": true` is typically equivalent to:
 If you however want to adapt any of the above default entries,
 you can make use of the more advanced options below.
 
+### Advanced: Using a new plugin
```

Comment:
If I understand correctly, this section talks about injecting components from another package into a given package.

Could you move this section to the end of this file, and create a new section on the same level as "Package.json discovery"? (`##`) Because it doesn't really fit into the package.json discussion IMO.

I would suggest renaming this section to something like "Injecting plugins across packages".